lib/Models/Catalog/Esri/ArcGisMapServerCatalogGroup.ts

```diff
@@ -49,6 +49,18 @@ export class MapServerStratum extends LoadableStratum(
     ) as this;
   }
 
+  findParentLayers(layerId: number): number[] {
```

Comment:
Can you please add a comment to highlight that it also returns the `layerId`, not just parentlayers?